deploy.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
```

Comment:
You are already logged in by your `before_script` stage, so this should be unnecessary